### DISCUSSION TITLE
Command to scale spot

### DISCUSSION THREAD

**Discussion by mondele** [jahnf/Projecteur#200] [Q&A]
I have Projecteur set up to toggle visibility of the spot (probably unnecessary, since it disappears with the next mouse click) with mouse button 4. I'm using `xbindkeys` for this. I love to be able to scale the spot up and down using a combination of a keypress and the mouse wheel, but all I see is setting the integer directly for the spot size. Am I missing a command to gradually increase or decrease the size of the spot?

**Comment by jahnf** [comment]
No you are right, there is no such command - the spot size can only be set in absolute values. The spot size is set as percentage of the screen height. (1-100 %)

**Comment by jahnf** [comment]
Feature request: #209
